depends/packages/boost.mk

```diff
@@ -14,6 +14,7 @@ $(package)_config_opts_debug=variant=debug
 $(package)_config_opts=--layout=system
 $(package)_config_opts+=threading=multi link=static -sNO_BZIP2=1 -sNO_ZLIB=1
 $(package)_config_opts_linux=threadapi=pthread runtime-link=shared
+$(package)_config_opts_freebsd=cxxflags=-fPIC
```

Comment:
This shouldn't be necessary; we set `$(package)_cxxflags_freebsd=-fPIC` below. (If that is not working, we should figure out why.)

Reply:
Without that line, the build [fails](https://github.com/zcash/zcash/files/5772223/build_error.txt).